templates/default/etc/init.d/node-ff.erb

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+#
+# chkconfig: - 80 20
+# processname: node-ff
+# description: Selenium Grid node for Firefox
+#
+
+### BEGIN INIT INFO
+# Provides: Selenium Grid Firefox node
+# Required-Start: $network $syslog
+# Required-Stop: $network $syslog
+# Default-Start:
+# Default-Stop:
+### END INIT INFO
+
+PATH_TO_JAR=<%= node['aet']['seleniumgrid']['node_ff']['root_dir'] %>/selenium-server-standalone-3.8.1.jar
+HUB_URL="http://localhost:4444"
+CAPABILITIES="browserName=firefox,version=ANY,platform=LINUX,maxInstances=5"
```

Comment:
Could we specify the version here?
We could set which version we are providing, but then when asking the hub:

* "capabilities that are not specified will be ignored"

https://github.com/SeleniumHQ/selenium/wiki/Grid2#using-grid-to-run-tests

Reply:
This idea makes sense only in case when we decide to support different versions of FF in the same time.

With current approach,  node `node-ff` will be available for any version of FF.
If we restrict to version 38 in cookbook now, then it will be available only for tests requests on 38. If we decide to upgrade to newer FF we will have  to upgrade cookbook also to handle newer version of FF.